parallel_algorithms/P3179/P3179.bs

```diff
@@ -812,7 +812,39 @@ an analog with very close semantics exists:
 
 ### The counterparts of exiting algorithms without `ExecutionPolicy` ### {#without_exec_policy_counterpart}
 
-// TODO: Add a full list
+Below we list the algorithms below without `ExecutionPolicy` in C++17 and where `ExecutionPolicy` parameter doesn't seem
+to add value:
+
+- `push_heap`
+- `pop_heap`
+- `sort_heap`
+- `next_permutation`
+- `prev_permutation`
+- `lower_bound`
+- `upper_bound`
+- `equal_range`
+- `binary_search`
+- `partition_point`
+- `sample` (RNG specific)
+- `shuffle` (RNG specific)
```

Comment:
Unlike all other algorithms in the list where we are pretty sure about no value from parallelism, I am not that sure to claim it for these two RNG based algorithms. There are articles about parallelizing random permutation (shuffle), and I guess that sampling can be in theory parallelized as well. For me, these are in the "investigation required" category.

Reply:
Move to "Possible" section